app/views/home/businesses.html.erb

```diff
@@ -103,15 +103,15 @@
       <h3 class="alt-h3">Set aside time</h3>
 
       <p class="text-gray mb-4">
-        Encourage your managers, employees, and coworkers to spend at least two hours on one day every week to work on the open source projects your business already uses.
+        Once a week, encourage your managers, employees, and coworkers to spend at least two hours working on the open source projects your organization already uses.
```

Comment:
Somewhat tangential: I'm thinking given the dashboard is only looking at Friday we should pivot the copy to say the same. Thoughts?